src/Files.App/Views/Properties/DetailsPage.xaml

```diff
@@ -91,14 +98,18 @@
 				<ListView.Footer>
 					<Button
 						x:Name="ClearPropertiesButton"
-						Margin="0,4,0,0"
+						Margin="0,8,0,0"
 						HorizontalAlignment="Stretch"
-						Content="{helpers:ResourceString Name=ClearPropertiesButton/Content}">
+						Content="{helpers:ResourceString Name=ClearPropertiesButton/Content}"
+						Style="{StaticResource EnhancedButtonStyle}">
 						<Button.Flyout>
 							<Flyout x:Name="ClearPropertiesFlyout">
-								<StackPanel Orientation="Vertical" Spacing="4">
+								<StackPanel Orientation="Vertical" Spacing="8">
 									<TextBlock Text="{helpers:ResourceString Name=ClearPropertiesFlyoutText/Text}" />
-									<Button Click="ClearPropertiesConfirmation_Click" Content="{helpers:ResourceString Name=Clear}" />
+									<Button
+										HorizontalAlignment="Right"
```

Comment:
Does this change the UI or is it just for readability?

Reply:
Before/After

![image](https://user-images.githubusercontent.com/62196528/232265838-61a7cd03-3d18-4933-b510-f619609e5ccd.png)

![image](https://user-images.githubusercontent.com/62196528/232265844-31da4965-9b6a-4b0d-aa09-1de9d4be2e0a.png)

Reply:
Much better! I wonder if we should also switch to teaching tip (in a separate PR).

Reply:
Yeah, of course.